src/components/Select/Select.tsx

```diff
@@ -141,6 +141,13 @@ const Select: React.FunctionComponent<React.PropsWithChildren<SelectProps>> = ({
     }
   }, [])
 
+  useEffect(() => {
+    if(defaultOptionIndex) {
```

Comment:
```suggestion
    if (defaultOptionIndex) {
```

do you have prettier or any editor extension installed?
recommended to have a proper setup for formatting

Reply:
fixed